app/controllers/nonsipp/schemedesignatory/HowManyMembersController.scala

```diff
@@ -141,9 +141,9 @@ object HowManyMembersController {
     Message("howManyMembers.heading", schemeName, endDate.show),
     TripleQuestion(
       form,
-      QuestionField.input("howManyMembers.field1", hint = Some("howManyMembers.field1.hint")),
-      QuestionField.input("howManyMembers.field2", hint = Some("howManyMembers.field2.hint")),
-      QuestionField.input("howManyMembers.field3", hint = Some("howManyMembers.field3.hint"))
+      QuestionField.numeric("howManyMembers.field1", hint = Some("howManyMembers.field1.hint")),
```

Comment:
Ticket (github) mentions "Check all inputs" and "If you’re asking the user to enter a whole number, set the inputmode attribute to numeric" 
Apart from this page, we have to user this new number questionField on all other pages where we are asking for a whole number.